components/events/GridEvents.jsx

```diff
@@ -0,0 +1,19 @@
+export default class GridEvents extends React.Component {
+
+  render() {
+    const { children }= this.props;
+    return(
+      <section className="gridEvents">
```

Comment:
Semantically speaking, having `ul` -> `li` elements here would be bring more accessibility.

Reply:
Absolutely you're right